#goal definition
float32[] x_setpoint
float32[] y_setpoint
float32[] z_setpoint
float32[] yaw_setpoint
float32[] linear_vel_max
float32[] angular_vel_max
float32[] anticipation
---
#result definition
bool finished
---
#feedback
float32 actual_x_sp
float32 actual_y_sp
float32 actual_z_sp
float32 actual_yaw_sp
float32 actual_vx_sp
float32 actual_vy_sp
float32 actual_vz_sp
float32 actual_yaw_rate_sp
